v1/api/sites.ts

```diff
@@ -83,33 +84,53 @@ export const siteRoutes = (store: StoreI) => async (server: FastifyTypebox): Pro
   }, async (request, reply) => {
     const { id } = request.params
     await store.sites.update(id, request.body)
-    return reply.code(200).send()
+    return await reply.code(200).send()
   })
 
+  async function processRequestFiles (request: FastifyRequest, reply: FastifyReply, fn: (filePath: string) => Promise<void>): Promise<void> {
+    try {
+      const files = await request.saveRequestFiles({
+        limits: {
+          files: 1,
+          fileSize: 1e9 // 1GB,
```

Comment:
@fauno How do you feel about this size limitation? Does that work for the average sutty site? Should we have a config somewhere where admins can increase it?

Reply:
@RangerMauve @jackyzha0 is there a need for a limit?  most sites are under 100MB, the biggest is 2.8GB, and some between 1-2 GB would be interesting to have on dweb, but they also embed youtube videos :(

Reply:
Yeah I just think it would be reasonable to set _some_ sort of limit (against people who may upload maliciously large payloads) and thought 1GB would be a good upperbound (most sites I own are ~10-30MB).